scripts/kallsyms.c

```diff
@@ -474,32 +474,32 @@ static void write_src(void)
 			markers[i >> 8] = off;
 
 		/*
-		 * There cannot be any symbol of length zero -- we use that
-		 * to mark a "big" symbol (and it doesn't make sense anyway).
+		 * There cannot be any symbol of length zero.
```

Comment:
One-line comments use this style:

```suggestion
		 /* There cannot be any symbol of length zero. */
```